grobid-core/src/main/java/org/grobid/core/sax/PDFALTOOutlineSaxHandler.java

```diff
@@ -53,11 +54,14 @@ public void endElement(java.lang.String uri, java.lang.String localName,
 			java.lang.String qName) throws SAXException {
 
 		if (qName.equals("STRING")) {
-			label = getText();
+		    currentNode.setLabel(getText());
+            currentNode.setBoundingBox(box);
```

Comment:
bounding box is not yet defines at the level of STRING, it's when ITEM is closed that we know for sure (see previous version)

Reply:
good point. I update this and include the bounding box in the text. However, if you check the test `testParsing_pdf2XMLOutline_errorcase_ShouldWork` the bounding box values (e.g. left and right seems to be inverted?)

Reply:
I didn't change the left and right attributes from pdf2xml, but at the time I was not 100% sure how to set them. I looked at some examples and right now it's:

```
                        double x = left;
			double y = right;
			double width = -1.0;
			double height = -1.0;
			if (right >= left)
				width = right - left;
			if (bottom >= top)
				height = bottom - top;
			box = BoundingBox
				.fromPointAndDimensions(page, x, y, width, height);
```

From your example, it's not working and it would be rather:  `top` is `y`, `bottom` is `height`, `left` is `x`, right is `width`. 
I am not sure it's consistent from one PDF to another one. Ideally it should be corrected in pdfalto (there's actually some piece of conversion into x/y in pdfalto for this, but it's finally not outputted :) ).

Reply:
Great! Thanks! I've updated the code to follow the new convention. 
I've also fixed another bug occurring when adding the bounding box at the end of the `<ITEM>` will not work for nested structure, I now add the bounding box right at the end of `<LINK>`

BTW: In the example also the pages don't seem consistent (lower items point to the first page), but it's a bug in the PDF.

Reply:
OK sorry I was not clear enough, the correspondence I indicated was only for your new example, it's going to clash with others. The way it was set first works for some documents and should be taken into account. In general I think it should be addressed in pdfalto considering all the different cases we can observe in a PDF.

Reply:
> BTW: In the example also the pages don't seem consistent (lower items point to the first page), but it's a bug in the PDF.

Yes this outline example has actually multiple bugs/problems, it should probably not be considered for general test :D

Reply:
> I've also fixed another bug occurring when adding the bounding box at the end of the `<ITEM>` will not work for nested structure, I now add the bounding box right at the end of `<LINK>`

Good point yes !

Reply:
OK. I reverted the change on the bounding box. 
I also remove the bounding box values from the test, as they don't make much sense at the moment.. 
Even the other test case for the outline that I have `pdfalto.xml_outline.xml` doesn't seem totally correct. 
Perhaps I will add an issue on PDFalto about it.